darwin/dataset/download_manager.py

```diff
@@ -339,13 +340,7 @@ def _download_single_slot_from_json_annotation(
             image = slot.source_files[0]
             image_url = image["url"]
             image_filename = image["file_name"]
-
-            if not use_folders:
-                suffix = Path(image_filename).suffix
-                stem = annotation_path.stem
-                filename = str(Path(stem + suffix))
-            else:
-                filename = slot.source_files[0]["file_name"]
+            filename = slot.source_files[0]["file_name"]
```

Comment:
Currently when pulling single-slotted items:
- If pulling without folders, we name the resulting files based on `item.name`
- If pulling with folders, we use `source_files[0].file_name`

`item.name` is the alias that the item has in Darwin. This can be changed by users, but `source_files[0].file_name` cannot. This change is a part of a push to unify how we name, save, & load single-slotted items

One thing to note about this change is that it's possible to register the same file from external storage (same `source_files[0].file_name`) as two differently named (`item.name`) dataset items in the same dataset folder. With these changes, pulling a dataset with these files would result in one overwriting the other **if** it was pulled without folder